app/views/users/show.html.erb

```diff
@@ -1 +1,26 @@
-<h1>Here is the info for a particular user</h1>
\ No newline at end of file
+<div class="user-show-page">
+  <div class="user-cont">
+    <div>
+      <img src="https://user-images.githubusercontent.com/37116322/222183427-3450696b-429a-4e9a-9e19-48bbe4b48edf.png" alt="profile">
+    </div>
+    <div class="user-info">
+      <h2><%= @user.name %></h2>
+      <div>
+        <p>Number of posts: <%= @user.posts_counter %></p>
+      </div>
+    </div>
+  </div>
```

Comment:
## [OPTIONAL]
- [ ] As you have already a partial defined in the above review comment, you will not need again to add this code above just call the partial `app/views/users/_user.html.erb`.

```html.erb
    <%= render @user %>